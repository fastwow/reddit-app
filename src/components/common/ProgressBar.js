import React from 'react';
import {
  View,
  ActivityIndicator,
} from 'react-native';
import styles from './styles';

const ProgressBar = () => (
  <View style={styles.progressBar}>
    <ActivityIndicator size="large" color={'#000000'}/>
  </View>
);

export default ProgressBar;
