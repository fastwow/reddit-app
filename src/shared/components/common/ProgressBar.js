import React from 'react';
import {ActivityIndicator, View} from 'react-native';
import styles from './styles';

const ProgressBar = () => (
  <View style={styles.progressBar}>
    <ActivityIndicator size="large" color={'#000000'}/>
  </View>
);

export default ProgressBar;
