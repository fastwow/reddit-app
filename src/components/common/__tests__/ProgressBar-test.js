import React from 'react';
import renderer from 'react-test-renderer';
import ProgressBar from '../ProgressBar';

test('ProgressBar renders correctly', () => {
  const tree = renderer.create(
    <ProgressBar/>,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
