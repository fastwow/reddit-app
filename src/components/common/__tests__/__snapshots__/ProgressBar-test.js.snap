// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProgressBar renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "height": 50,
      "justifyContent": "center",
    }
  }
>
  <ActivityIndicator
    animating={true}
    color="#000000"
    hidesWhenStopped={true}
    size="large"
  />
</View>
`;
